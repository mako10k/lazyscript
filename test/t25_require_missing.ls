!{ ~~require "no_such_module_ABCDEF.ls"; ~~println "after" };
