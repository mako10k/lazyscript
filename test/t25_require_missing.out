E: <bottom msg="require: not found" at <unknown>:1.1: >
